src/react/header/MODlogo.tsx

```diff
@@ -0,0 +1,17 @@
+export const MODlogo = () => (
```

Comment:
I wonder if we can use https://github.com/airbnb/babel-plugin-inline-react-svg and share the SVG with the Nunjucks macro?